pkgs/by-name/st/stereotool/package.nix

```diff
@@ -0,0 +1,162 @@
+      })
+    ];
+    # Sources if the system is aarch64-linux
+    aarch64-linux = [
+      (fetchurl {
+        name = "alsa";
+        url = "https://download.thimeo.com/stereo_tool_gui_pi2_64_${versionNoPoint}";
+        hash = "sha256-iwoc6c+ox+2DSqmiz8mpDotDjqki7iL0jgqc7Z1htNI=";
+      })
+      (fetchurl {
+        name = "jack";
+        url = "https://download.thimeo.com/stereo_tool_gui_jack_pi2_64_${versionNoPoint}";
+        hash = "sha256-iwoc6c+ox+2DSqmiz8mpDotDjqki7iL0jgqc7Z1htNI==";
+      })
+      (fetchurl {
+        name = "cmd";
+        url = "https://download.thimeo.com/stereo_tool_pi2_64_${versionNoPoint}";
+        hash = "sha256-bIFnQkJB9XoEKo7IG+MSMvx/ia1C8i97Cw7EX4EDizk=";
+      })
+    ];
+    # Sources if the system is aarch32-linux
+    aarch32-linux = [
+      (fetchurl {
+        name = "alsa";
+        url = "https://download.thimeo.com/stereo_tool_gui_pi2_${versionNoPoint}";
+        hash = "sha256-922yqmis5acvASU2rEi5YzFLAUuDO7BiEiW49RKfcoU=";
+      })
+      (fetchurl {
+        name = "jack";
+        url = "https://download.thimeo.com/stereo_tool_gui_jack_pi2_${versionNoPoint}";
+        hash = "sha256-922yqmis5acvASU2rEi5YzFLAUuDO7BiEiW49RKfcoU=";
+      })
+      (fetchurl {
+        name = "cmd";
+        url = "https://download.thimeo.com/stereo_tool_pi2_${versionNoPoint}";
+        hash = "sha256-xKM5Mg6gEAvbp63rd81ssnx2Bj1hUylCo36mQBYwIvg=";
+      })
+    ];
+    # Sources if the system is 32bits i686
+    i686-linux = [
+      (fetchurl {
+        # The name is the name of this source in the build directory
+        name = "alsa";
+        url = "https://download.thimeo.com/stereo_tool_gui_${versionNoPoint}";
+        hash = "sha256-iEPqJvmXKXD4AVbM+1QZeUOwpMjMT7ROYNQpmhRVZyw=";
+      })
+      (fetchurl {
+        name = "jack";
+        url = "https://download.thimeo.com/stereo_tool_gui_jack_${versionNoPoint}";
+        hash = "sha256-iEPqJvmXKXD4AVbM+1QZeUOwpMjMT7ROYNQpmhRVZyw=";
+      })
+      (fetchurl {
+        name = "cmd";
+        url = "https://download.thimeo.com/stereo_tool_cmd_${versionNoPoint}";
+        hash = "sha256-sk13wj7XvuwTDWWW6tMYHdTV9XjPeHe6hHv2JPBxBLA=";
+      })
+    ];
+  }.${stdenv.hostPlatform.system} or (throw "Unsupported system: ${stdenv.hostPlatform.system}");
+
+  # Just copy all sources in a single source folder:
+  # https://discourse.nixos.org/t/mkderivation-src-as-list-of-filenames/3537/10
+  unpackPhase = ''
+    for srcFile in $srcs; do
+      cp $srcFile $(stripHash $srcFile)
+    done
+  '';
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    makeWrapper
+    copyDesktopItems
+  ];
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = "${pname}-alsa";
+      desktopName = "Stereotool-Alsa";
+      exec = "stereo_tool_gui";
+      icon = "stereo-tool-icon";
+      comment = "Broadcast Audio Processing";
+      categories = [ "AudioVideo" "Audio" "AudioVideoEditing" ];
+    })
+    (makeDesktopItem {
+      name = "${pname}-jack";
```

Comment:
```suggestion
      name = "stereotool-alsa";
      desktopName = "Stereotool-Alsa";
      exec = "stereo_tool_gui";
      icon = "stereo-tool-icon";
      comment = "Broadcast Audio Processing";
      categories = [ "AudioVideo" "Audio" "AudioVideoEditing" ];
    })
    (makeDesktopItem {
      name = "stereotool-jack";
```

Reply:
Out of curiosity, why is this better?

Reply:
https://github.com/NixOS/nixpkgs/issues/277994